apps/src/templates/courseOverview/AssignToSection.js

```diff
@@ -49,7 +49,7 @@ class AssignToSection extends Component {
   updateCourse = () => {
     const section = this.props.sectionsInfo[this.state.sectionIndexToAssign];
     $.ajax({
-      url: `/sections/${section.id}`,
+      url: `/dashboardapi/sections/${section.id}`,
```

Comment:
is it safe to update both the front end and backend at the same time? Dave was seeing a bunch of errors earlier this week because users has old JS code, but the server had new code.

Reply:
that's a good question. i don't believe i've updated both at once before. @islemaster do you have more insight into this?

Reply:
I would expect a brief window of JavaScript errors where clients try to hit the old API and get 404s.  The negative user impact of this is probably that nothing happens, and they need to reload the page to get things working again.

I think we cause this sort of issue all the time and it's probably not a big deal. If you wanted to be super-careful, you could add the new API and update the consumers here, but not tear out the old API until a subsequent DTP.

Reply:
oh, i see. just to be safe i'll make the parallel change here then make a separate PR to remove the old endpoint and swap consumers to the new endpoint